Sources/AppCoinsSDK/UI/Purchase/ViewModels/TransactionViewModel.swift

```diff
@@ -165,7 +166,7 @@ internal class TransactionViewModel : ObservableObject {
         }
     }
     
-    private func getWalletBalance(wallet: Wallet, currency: Coin, completion: @escaping (Balance) -> Void) {
+    private func getWalletBalance(wallet: Wallet, currency: String, completion: @escaping (Balance) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done